src/porepy/params/data.py

```diff
@@ -50,6 +50,17 @@
 bucket), consider:
 
     pp.initialize_default_data(grid, d, keyword, specified_parameters)
+
+
+Also contains a function for setting the state. The state is all data associated with
+the previous time step or iteration, and is stored in data[pp.STATE]. The solution of a
+variable is stored in
+
+data[pp.STATE][variable_name],
+
+whereas data such as BC values are stored similarly to in the Parameters class, in
+
+data[pp.STATE][keyword]["bc_values"].
```

Comment:
Maybe
data[pp.STATE][variable_name]["values"] and
data[pp.STATE][variable_name]["bc_values"] ??
or:
data[pp.STATE][keyword][variabel_name] and
data[pp.STATE][keyword]["bc_values"] ??

or: 
data[pp.STATE][variable_name]
data[pp.STATE][bc_values_name] ??